Our rust doesn't like LTO due to us using LLVM 7

--- Cargo.toml
+++ Cargo.toml
@@ -40,7 +40,3 @@ gettext-rs = { version="0.4", features=["gettext-system"] }
 crossbeam-deque = "0.7"
 cairo-sys-rs = {git="https://github.com/gtk-rs/cairo"}
 human-panic = "1"
-
-[profile.release]
-lto = true
-codegen-units = 1
